src/App.js

```diff
@@ -11,7 +12,10 @@ import 'antd/dist/antd.css';
  */
 function App() {
   return (
-      <Header />
+    <div className='app-container'>
+          <Header />
```

Comment:
The indentation in component files will be two or four spaces? It's not consistent in this file.

Reply:
I updated the entire project to use 4.